core/debug/src/debug.c

```diff
@@ -30,6 +30,41 @@ uint8_t g_buffer[DEBUG_MAX_BUFFER];
 int g_buffer_pos;
 
 #ifndef CHANNEL_DEBUG
```

Comment:
@fvincenzo please move all the beetle-specific code to a dedicated debug_beetle.c file.